tests/ignite/engine/test_create_supervised.py

```diff
@@ -93,7 +96,7 @@ def _test_create_supervised_evaluator(
         example_input = torch.randn(1, 1)
         model = torch.jit.trace(model, example_input)
 
-    evaluator = create_supervised_evaluator(model, device=evaluator_device)
+    evaluator = create_supervised_evaluator(model, device=evaluator_device, amp_mode=amp_mode)
```

Comment:
Could you please check that `evaluator._process_function` is `supervised_evaluation_step` or `supervised_evaluation_step_amp`

Reply:
@Moh-Yakoub could you please the above check here and we can merge this PR ?

Reply:
@vfdev-5 sure, adding it now.